### PR TITLE
make: add python3 to test with both versions

### PR DESCRIPTION
**Pull request by JanZerebecki**
Switch to Ubuntu Xenial for python3-libvirt which wasn't available in
Trusty.

Also:
- py23: use string.ascii_lowercase
instead of string.lowercase, as the later is not available in python 3.

### CODE REVIEW

**Comment by nicolasbock** [comment]
The failure in Travis would be fixed with #3424

File: Makefile [hunk 1]
```diff
@@ -71,13 +73,14 @@ install: debianinstall genericinstall
 
 debianinstall:
 	sudo apt-get update -qq
-	sudo apt-get -y install libxml-libxml-perl libjson-perl libjson-xs-perl python-libvirt
+	sudo apt-get -y install libxml-libxml-perl libjson-perl libjson-xs-perl libwww-perl python-pip python3-pip python-setuptools python3-setuptools python-libvirt python3-libvirt
 
 suseinstall:
-	sudo zypper install perl-JSON-XS perl-libxml-perl python-pip libvirt-python
+	sudo zypper install perl-JSON-XS perl-libxml-perl python-pip python3-pip libvirt-python python3-libvirt-python
```
Comment (bmwiedemann) [inline Makefile]: if we add `libwww-perl` above, we should also add perl-CGI here

Comment (JanZerebecki) [reply Makefile]: Done: I assume you meant: `perl-libwww-perl`

**Comment by stefannica** [comment]
Would be nice to have travis python tests also cover the scripts/jenkins/ardana/gerrit folder. @nicolasbock , are you already working on this ?

**Comment by nicolasbock** [comment]
No, I haven't looked at the `gerrit` folder. But I am working on unit testing the `ansible` folder.

**Pull request merged**